manifests/mod/php.pp

```diff
@@ -49,7 +50,7 @@
     $_package_name = undef
   }
 
-  $_lib = "libphp${php_version}.so"
+  $_lib = "$libphp_prefix${php_version}.so"
```

Comment:
This should be using braces: `${libphp_prefix}`

Reply:
I have now added the braces.